lib/common/utils.js

```diff
@@ -75,7 +77,13 @@ export const getErrorParameters = function (args) {
     stack = isErrorObject && error.stack || createStackTrace();
     type = options.type;
     subType = options.subType;
+    kadiraInfo = options.kadiraInfo;
   }
 
-  return { type, message, subType, stack };
+  return { type, message, subType, stack, kadiraInfo };
+};
+
+
+export const objectHasData = function (obj) {
+  return Object.values(obj).some(Boolean);
```

Comment:
It might be good to add a comment or indicate in the function name that this checks if the object has truthy data (it returns false even if the object has `false` values which is correct, but could be unexpected).

Reply:
That is a fair point, actually I think it is best that fix that behavior altogether.